src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -29,10 +29,6 @@ public static class LanguageServerConstants
 
         public const string RazorCodeActionRunnerCommand = "razor/runCodeAction";
 
-        public const string RazorDocumentFormattingEndpoint = "textDocument/formatting";
-
-        public const string RazorDocumentOnTypeFormattingEndpoint = "textDocument/onTypeFormatting";
```

Comment:
These were using the per-spec LSP methods, but for internal comms between our language servers. Bad mojo. This is where I started because I assumed this would be the issue, but I was totally wrong, but figured it was still good to fix this. Most of the changes are due to this clean up :)